torch/_inductor/codegen/cpp.py

```diff
@@ -2062,9 +2059,36 @@ def add_to_dtype(sub_graph: torch.fx.Graph):
                     pass
 
             def eliminate_to_dtype(sub_graph: torch.fx.Graph):
-                # TODO(Eikan) Remove redundant to_dtype like load_bf16 + to_fp32 + to_bf16 + store_bf16
-                # => load_bf16 + store_bf16
```

Comment:
The new function removes the consecutive to_dtype with the same dtype. Are you also going to cancel the "to_fp32 + to_bf16" as illustrated in the comment above?

Reply:
Move this feature to `_eliminate_to_node_mem_copy`.